hw/ip/dcd/data/dcd.hjson

```diff
@@ -0,0 +1,402 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "dcd",
+  clock_primary: "clk_i",
+  other_clock_list: [
+    "clk_aon_i",
+  ],
+  reset_primary: "rst_ni",
+  bus_device: "tlul",
+  bus_host: "none",
+  //inter_signal_list: [
```

Comment:
why comment this out..?

Reply:
I didn't see any effect on the generated CSR files. Hence, playing with inter_signal_list.

Reply:
o the inter-signal is used during integration, it won't change the module itself. So it's best to keep it there.